src/nodedef.h

```diff
@@ -157,12 +159,13 @@ enum NodeDrawType
 	NDT_FENCELIKE,
 	NDT_RAILLIKE,
 	NDT_NODEBOX,
-	NDT_GLASSLIKE_FRAMED, // Glass-like, draw connected frames and all all
-	                      // visible faces
-						  // uses 2 textures, one for frames, second for faces
+	NDT_GLASSLIKE_FRAMED, // Glass-like, draw connected frames and all visible faces
+	                      // param2 > 0 defines 64 levels of internal liquid
+						  // Uses 3 textures, one for frames, second for faces,
+	                      // optional third is a 'special tile' for liquid.
 	NDT_FIRELIKE, // Draw faces slightly rotated and only on connecting nodes,
-	NDT_GLASSLIKE_FRAMED_OPTIONAL,	// enabled -> connected, disabled -> Glass-like
-									// uses 2 textures, one for frames, second for faces
+	NDT_GLASSLIKE_FRAMED_OPTIONAL,	// Enabled -> connected, disabled -> Glass-like
+									// Uses 2 textures, one for frames, second for faces
```

Comment:
Yes multiple tabs have appeared. Somehow my editor must have changed my multiple spaces into tabs. I'll reformat the whole section to avoid these mixes of tabs and spaces.